manifests/a/ApacheFriends/Xampp/8.1.5-0/ApacheFriends.Xampp.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: ApacheFriends.Xampp
+PackageVersion: 8.1.5-0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: --mode unattended --launchapps 0
+  SilentWithProgress: --mode unattended --launchapps 0
+UpgradeBehavior: install
+ReleaseDate: 2022-04-20
+Installers:
+- Architecture: x64
+  InstallerUrl: https://sourceforge.net/projects/xampp/files/XAMPP Windows/8.1.5/xampp-windows-x64-8.1.5-0-VS16-installer.exe/download
```

```suggestion
  InstallerUrl: https://downloadsapachefriends.global.ssl.fastly.net/8.1.5/xampp-windows-x64-8.1.5-0-VS16-installer.exe
```
This is the domain that all of the other versions that I checked are using.  This file exists and has the same hash.